### DISCUSSION TITLE
ILS in radnav

### DISCUSSION THREAD

**Discussion by terrainahead** [flybywiresim/aircraft#2975] [Help]
With the latest custom FBW 10/01/2021 build the ILS frequency does not show in the navrad page after selecting the arrival. I wait till I am near the airport to verify.
I have to insert it manually.
Can I confirm if this is normal? The FBW is amazing.
Many thanks for all the hard work
Shane

**Comment by Inaxair** [comment]
Hello @terrainahead 

Can you be more specific? What are the routes, steps, and way you did to achieve this issue?

Best regards,
Inaxair from FlyByWire Simulations

**Comment by Elmarqo** [comment]
Hello,
I encountered the same issue. There is no frequency of ILS after putting transition or STAR in. I met it at LOWW RWY16, EPWR RW29 and others.  To land safely I had to set frequency manually before landing.
